<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogFileScreen</class>
 <widget class="QDialog" name="dialogFileScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dialogFileScreen</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Save screen or sequence of screens to JPEG file(s)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Size of saved image in x</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>and y</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Single image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>output filename</string>
   </property>
  </widget>
  <widget class="QPushButton" name="oneBrowseButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="oneSnapButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Snap Single</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="oneBrowseEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Multiple images</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>e.g. base name 'C:/FusionDd/bin/temp' would produce a sequence</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>'C:/Fusion3D/bin/temp0001.jpg', 'C:/Fusion3D/bin/temp.0002.jpg' ...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Base name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multBrowseButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="multBrowseEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="snapNextButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Snap Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="xEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>oneBrowseButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogFileScreen</receiver>
   <slot>doOneBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>multBrowseButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogFileScreen</receiver>
   <slot>doMultBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oneSnapButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogFileScreen</receiver>
   <slot>doOneSnap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>snapNextButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogFileScreen</receiver>
   <slot>doMultSnap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogFileScreen</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doOneBrowse()</slot>
  <slot>doMultBrowse()</slot>
  <slot>doOneSnap()</slot>
  <slot>doMultSnap()</slot>
 </slots>
</ui>
